src/migration/1727345240782-SourceFlagsIndexes.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class SourceFlagsIndexes1727345240782 implements MigrationInterface {
+  name = 'SourceFlagsIndexes1727345240782';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE INDEX "IDX_source_flags_posts_members_threshold" ON source(((flags->'totalPosts')::integer), ((flags->'totalMembers')::integer), ((flags->'publicThreshold')::boolean))`,
+    );
+    await queryRunner.query(
+      `UPDATE "public"."source" SET flags = flags || '{"publicThreshold": false}' WHERE type = 'squad'`,
```

Comment:
Should we not set everything to true for now?

Reply:
We discussed to update this a bit in the earlier phase. I guess it is better to put the right value now rather than wait for the cron job.

Reply:
Updated now, would love to hear what you think.